### DISCUSSION TITLE
Optimising your WP_Query queries for better performance

### DISCUSSION THREAD

**Discussion by bph** [WordPress/developer-blog-content#27] [Topic Ideas]
Thank you to @zzap for suggesting this topic. 

Can you think of an outline for a blog post? What specifically should be covered?

**Comment by ryanwelcher** [comment]
[This support thread](https://wordpress.org/support/topic/internal-link-search/#post-16407817) tangentially ran into this issue. It would be great to have an official post on this as I had to point them to the [10up Best Practices](https://10up.github.io/Engineering-Best-Practices/php/#performance) for examples.

**Comment by leac** [comment]
Delicious Brains posted about this topic just a few days ago. Does that have any impact on this suggestion?
https://deliciousbrains.com/sql-query-optimization-for-faster-wordpress-sites/

**Comment by bph** [comment]
@Kadamwhite talk about it in context of the REST API, too at WordCamp Asia. It might be out of scope for this context. 
[Getting](https://asia.wordcamp.org/2023/session/getting-the-most-out-of-the-rest-api/)[ the most out of the REST API](https://asia.wordcamp.org/2023/session/getting-the-most-out-of-the-rest-api/)
with [K.Adam White](https://asia.wordcamp.org/2023/speaker/k-adam-white/)

White renamed the talk to **“Getting the most out of WordPress, when you’re writing Block Editor code”**.

[Recording](https://www.youtube.com/live/vrDwB7iuHL8?t=2299) | [Slidedeck](https://docs.google.com/presentation/d/1PQVcTzcUP2xRbwjEjqo8VENxU7fiDBtz8U7R4cA0-IE/edit#slide=id.g1f27001dabd_0_8)

**Comment by bph** [comment]
Conversation locked. Please continue discussion on the issue 
- #128
